package.json

```diff
@@ -184,12 +184,14 @@
             {
                 "command": "zeplin.sidebar.jumpTo",
                 "key": "ctrl+k j",
-                "mac": "cmd+k j"
+                "mac": "cmd+k j",
+                "when": "!terminalFocus"
```

Comment:
Rather than blacklisting here, can we activate these commands when focused view is our extension using `focusedView` context variable? Because any other plugin might be using these shortcuts too and we'll block them as well.

Reply:
We want it to be global, not only when our view is active.

"Terminal: Clear" is the only built-in command that uses `cmd+k` shortcut afaik, `cmd+k plus a second key` is used by many other commands, so extension writers should know not use only `cmd+k` as a shortcut which means we should not have a problem by having a globally active `cmd+k plus a second key` shortcut